libs/ui-lib/lib/ocm/components/clusterDetail/ClusterSummaryExpandable.tsx

```diff
@@ -2,8 +2,8 @@ import { ExpandableSection } from '@patternfly/react-core';
 import React, { useState } from 'react';
 import { Cluster } from '../../../common';
 import { ReviewSummaryContent } from '../clusterConfiguration/review/ReviewSummary';
-import { ClusterFeatureSupportLevelsDetailItem } from '../featureSupportLevels';
 import './ClusterSummaryExpandable.css';
+import ClusterFeatureSupportLevelsDetailItem from '../newFeatureSupportLevels/ReviewClusterNewFeatureSupportLevels';
```

Comment:
Looks like parts of this MR is covered by https://github.com/openshift-assisted/assisted-installer-ui/pull/2244

Could that be reverted so this MR only handles the flag it's deprecating?